integration_tests/suite/test_http_callback.py

```diff
@@ -262,20 +273,29 @@ def test_given_one_http_subscription_when_bus_event_then_one_http_callback_that_
         ), tries=10, interval=0.5)
 
         webhookd = self.make_webhookd(MASTER_TOKEN)
-        logs = webhookd.subscriptions.get_logs(subscription["uuid"])
+        logs = webhookd.subscriptions.get_logs(subscription["uuid"],
+                                               direction="asc")
         self.assertEqual(2, logs['total'])
 
-        self.assertEqual("failure", logs['items'][0]["status"])
-        self.assertIn("Service Unavailable", logs['items'][0]["detail"]["error"])
-        self.assertEqual(1, logs['items'][0]["attempts"])
-
-        self.assertEqual("success", logs['items'][1]["status"])
-        self.assertEqual(2, logs['items'][1]["attempts"])
-        self.assertEqual('GET', logs['items'][1]["detail"]['request_method'])
-        self.assertEqual(
-            'http://third-party-http:1080/test?'
-            'test_case=test_given_one_http_subscription_when_bus_event_then_one_http_callback_that_return_500',
-            logs['items'][1]["detail"]['request_url'])
+        assert_that(logs['total'], 2)
+        assert_that(logs['items'], only_contains(
```

Comment:
Should be contains or contains_inanyorder. Because only_contains will not raise if result has the same item twice

Reply:
Good catch, in fact I always want the exact match contents + order, so I used `contains`.